snapcraft.yaml

```diff
@@ -0,0 +1,19 @@
+name: tqdm
+version: '1.0'
+summary: A fast, extensible progress bar for Python and CLI
+description: |
+  A fast, extensible progress bar for Python and CLI
+grade: stable
+confinement: strict
+base: core18
+icon: tqdm-logo.png
+parts:
+  tqdm:
+    plugin: python
+    python-version: python2
+    source: https://github.com/tqdm/tqdm.git
```

Comment:
is the `.git` extension required?

Reply:
Your source can be anything - online repos, archives, local files. As long as the source is python so the python plugin can build the application correctly.